dds/DCPS/XTypes/DynamicData.cpp

```diff
@@ -2193,6 +2239,458 @@ const char* DynamicData::typekind_to_string(TypeKind tk) const
   }
 }
 
+bool print_dynamic_data(DynamicData dd, DCPS::String& type_string, DCPS::String indent)
+{
+  OpenDDS::DCPS::String member_name;
```

Comment:
```suggestion
  DCPS::String member_name;
```

And for the rest of the function.